distributed/client.py

```diff
@@ -828,7 +828,10 @@ def dashboard_link(self):
             return self.cluster.dashboard_link
         except AttributeError:
             scheduler, info = self._get_scheduler_info()
-            protocol, rest = scheduler.address.split("://")
+            try:
+                protocol, rest = scheduler.address.split("://")
+            except AttributeError:
+                return None
```

Comment:
Nitpick: I think an explicit `scheduler is None` check here would be more clear. With `try`/`except AttributeError`, I'm not sure whether `scheduler.address` or `address.split("://")` is where the `AttributeError` is expected to come from.

```suggestion
            if scheduler is None:
                return None
            else:
                protocol, rest = scheduler.address.split("://")
```